### DISCUSSION TITLE
Using ponder like vite/snowpack?

### DISCUSSION THREAD

**Discussion by roninjin10** [ponder-sh/ponder#147] [General]
Is it possible to use ponder kinda l like vite where we use ponder for dev server but it still works deploying as a subgraph?   How heavy of a lift would it be to wrap ponder in an abstraction that does this?

**Comment by typedarray** [comment]
No, not without a big lift. I think at this point, ponder has diverged enough from subgraph APIs/semantics that it would be quite difficult to attempt to build ponder event handler files into subgraph-compatible WASM. I looked into this early on, but decided it was best to design new TypeScript-native APIs that take advantage of modern TS development tools.
